lib/transaction.js

```diff
@@ -14,35 +14,12 @@ const { uniqueId, isUndefined } = require('lodash');
 // Acts as a facade for a Promise, keeping the internal state
 // and managing any child transactions.
 class Transaction extends EventEmitter {
-  constructor(client, container, config, outerTx) {
+  constructor(client, container, config = {}, outerTx = null) {
     super();
+    this.doNotRejectOnRollback = config.doNotRejectOnRollback;
```

Comment:
Doesn't this change default value to `false` from current `true`?

Reply:
Yes and No.  It appears that `Transaction` construction is always kicked off from `lib/util/make-knex.js` .  The logic for handling defaults has been extracted to that location.

But... to be on the safe side, we might also want to provide an ULTRA-default value within the `Transaction` constructor as well.

Reply:
Right, I see, you handle default values elsewhere now.